addons/project/views/project_project_views.xml

```diff
@@ -219,7 +219,7 @@
                     <field name="last_update_color" column_invisible="True"/>
                     <field name="tag_ids" widget="many2many_tags" options="{'color_field': 'color'}" optional="hide"/>
                     <field name="last_update_status" string="Status" nolabel="1" optional="show" widget="project_state_selection" options="{'hide_label': True}"/>
-                    <field name="stage_id" options="{'no_open': True}" optional="show"/>
+                    <field name="stage_id" options="{'no_open': True}" domain="[('company_id', 'in', (company_id, False))]" optional="show"/>
```

Comment:
same comment here. 🙂

Reply:
it is the same here :slightly_smiling_face: